app/lib/graphql-api.js

```diff
@@ -0,0 +1,37 @@
+// TODO: Use ApolloJS (https://www.apollographql.com/docs/react/) to generate GraphQL requests.
+
+class Api {
+  static headers() {
+    return {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      'dataType': 'json',
+    }
+  }
+
+  static get(route) {
+    return this.xhr(route, null, 'GET');
+  }
+
+  static post(route, params) {
+    return this.xhr(route, params, 'POST');
+  }
+
+  static xhr(route, params, verb) {
+    // const host = 'https://api.uni.ninja'
```

Comment:
Can you please change this so that the `host` is the production ready endpoint?

Reply:
Done ✅